ocs_ci/ocs/ui/page_objects/infra_health.py

```
@@ -326,6 +326,51 @@ def unsilence_alerts(self):
+        Args:
+            alert_check_name (str): Name of the alert check to disable
+        """
+        logger.info(f"Disabling alert: {alert_check_name}")
+        alerts = self.filter_by_name_or_details(alert_check_name)
+        count = 0
+        for alert in alerts:
+            if alert.check == alert_check_name and alert.end_time is None:
+                self.do_click(
+                    format_locator(self.validation_loc["select_alert"], str(count))
+                )
+            count += 1
```

Ruff (0.15.15)

[warning] Use `enumerate()` for index variable `count` in `for` loop

(SIM113)